hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/EquivalenceFeature.java

```diff
@@ -82,59 +87,67 @@ public void selfDestructAndSetBeneficiary(String beneficiary) {
         }
     }
 
-    @Then("I execute balance opcode to system account {string} address would return 0")
-    public void balanceOfAddress(String address) {
-        var accountId = new AccountId(extractAccountNumber(address)).toSolidityAddress();
-        var parameters = new ContractFunctionParameters().addAddress(accountId);
-        var functionResult = executeContractCallQuery(deployedEquivalenceCall, "getBalance", parameters);
+    @Then("I execute balance opcode to system account {string} address would return 0 with call to {string}")
+    public void balanceOfAddress(String address, String node) {
```

Comment:
I meant use `@ParameterType` for automatic conversion. But can adapt in a follow up.
```suggestion
    @Then("I execute balance opcode to system account {string} address would return 0 with call to {node}")
    public void balanceOfAddress(String address, NodeNameEnum node) {
```